### PR TITLE
firewall: document ingressPolicy=(open|same-bridge)

### PR DESCRIPTION
**Pull request by AkihiroSuda**
The ingressPolicy parameter was implemented in https://github.com/containernetworking/plugins/pull/584 .

Merged into the master on Feb 10, 2022: https://github.com/containernetworking/plugins/commit/42268a4a93f6425413d4c840aa8717496ced12a0 (v1.1.0)

### CODE REVIEW

File: content/plugins/current/meta/firewall.md [hunk 1]
```diff
@@ -154,3 +154,48 @@ The chain name `CNI-ADMIN` can be overridden by specifying `iptablesAdminChainNa
     "iptablesAdminChainName": "SOME-OTHER-CHAIN-NAME",
 }
 ```
+
+## Ingress Policy
+
+Starting with vX.X.X (TBD), the `firewall` plugin supports `ingressPolicy` for isolating bridges.
```
Comment (AkihiroSuda) [inline content/plugins/current/meta/firewall.md:160]: Needs help to determine X.X.X

**Comment by squeed** [comment]
@AkihiroSuda the version is v1.1 :)

File: content/plugins/current/meta/firewall.md (hunk 1 above)
Comment (AkihiroSuda) [reply content/plugins/current/meta/firewall.md]: Updated to v1.1.0

**Comment by AkihiroSuda** [comment]
> @AkihiroSuda the version is v1.1 :)

Thanks, updated the PR for v1.1

**Pull request merged**